src/components/Alerts/StyledAlert.tsx

```diff
@@ -0,0 +1,127 @@
+import { ReactNode } from "react"
+import styled, { css, keyframes } from "styled-components"
+import { AlertContainerProps, AlertContentWrapperProps } from "./AlertInterface"
+import {
+  Magenta80,
+  Yellow80,
+  Green80,
+  Crimson80,
+  Neutral10,
+  Neutral0,
+} from "../../common/styles/palette"
+import Icon from "../Icon/index"
+
+const getWidth = (size: string) => {
+  switch (size) {
```

Comment:
width and background are static, so you can define objects beforehand and refer them directly instead of functions having switch cases. This will make it a bit less verbose.

```javascript
   const widthForSize = {
      "x-small": 280,
      ...
   }
```